test/powershell/Modules/Microsoft.PowerShell.Management/Set-Service.Tests.ps1

```diff
@@ -111,35 +111,39 @@ Describe "Set/New-Service cmdlet tests" -Tags "Feature", "RequireAdminOnWindows"
     }
 
     It "Set-Service can change credentials of a service" -TestCases @(
-        @{name = "testsetcredential"; 
-            startCredential = [System.Management.Automation.PSCredential]::new(".\Guest", 
-                (ConvertTo-SecureString "PlainTextPassword" -AsPlainText -Force)); 
-            endCredential = [System.Management.Automation.PSCredential]::new(".\Administrator", 
-                (ConvertTo-SecureString "PlainTextPassword" -AsPlainText -Force))}
+        @{name = "testsetcredential"; startUsername = "user1"; endUsername = "user2"}
     ) {
-        param($name, $startCredential, $endCredential)
+        param($name, $startUsername, $endUsername)
         try {
+            $testPass = "Secret123!"
+            $null = net user $startUsername $testPass /add
+            $null = net user $endUsername $testPass /add
```

Comment:
Minor comment - we usually use the pattern  `net user $startUsername $testPass /add > $null`